### PR TITLE
Update eslint rules to prevent let when const can be used

### LINKED ISSUE #38: Update eslint rules to prevent let, when const will do
**Is your feature request related to a problem? Please describe.**
We can write `let` instead of `const` currently, but this isn't neccessary.

**Describe the solution you'd like**
Update eslint rules to prevent using `let` if a `const` will do.

**Additional context**
Docs on possible solution:
https://eslint.org/docs/rules/prefer-const

### PR DESCRIPTION
**Pull request by Twaha-Rahman**
This PR closes #38

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/src/tests
```
Comment (bradtaniguchi) [inline .eslintignore]: tests should still be covered though right?

Comment (Twaha-Rahman) [reply .eslintignore]: ESLint goes crazy when it sees we are using Jest functions without importing it... that's why I had to ignore it

Comment (Ryuno-Ki) [reply .eslintignore]: Was there no way to declare it as a global?

Comment (bradtaniguchi) [reply .eslintignore]: I looked into this and found that we can remove the `.eslintignore` file, so we get linting in our tests. 
To add eslint support for jest all we need to do is update the `eslintrc.js` file's `env` property like so:
```js
module.exports = {
env: {
  //..
  jest: true
},
//..
```

[ref](https://stackoverflow.com/questions/31629389/how-to-use-eslint-with-jest)

Comment (Twaha-Rahman) [reply .eslintignore]: Ok... updating the PR!

**Comment by Twaha-Rahman** [comment]
@bradtaniguchi Done! I think we're ready to go!

**Pull request merged**